debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.5.100-15) unstable; urgency=medium
```

Comment:
This will conflict now, but so will my pydaos/ftest one and I'm hoping that can land in the next day or so.

Reply:
This can go after yours